pages/posts/hello-2023--hello-json-schema-community.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Hello 2023, Hello JSON Schema Community"
+date: 2023-01-10
+tags:
+  - Update
+  - News
+type: Community
+cover: /img/posts/2023/hello-2023--hello-json-schema-community/jai.webp
+authors:
+  - name: Benjamin Granados
+    photo: /img/avatars/benjagm.png
+    link: https://twitter.com/benjagm
+    byline: Developer Advocate
+---
+
+Since Ben Hutton joined Postman to work full-time on JSON Schema [the past year](https://blog.postman.com/ben-hutton-joins-postman-to-lead-json-schema-strategy/), other top contributors like Jason Desrosiers, Julian Berman, and Greg Dennis have joined, and now all are full-time supporting JSON Schema. So today, I am excited to announce that I am joining as Developer Advocate with a focus on serving the Community, nurturing relationships with other organizations, launching programs to support and grow the Community, and increasing awareness through content, public speaking, and discussions in Twitter, Linkedin, Stackoverflow or Reddit.
+
+## Who is this guy?
+
+My whole career, APIs have been at the core of everything. In the first part of my career, I focused on the engineering aspect of building APIs, playing all the possible team roles. Later, I focused on API Design and Governance, leading API Programs, and finally, my focus has been Developer Experience and Developer Relations because this human aspect is the most critical for a technology or product to be successful.
```

Comment:
I think the intention at the end here is:

```suggestion
My whole career, APIs have been at the core of everything. In the first part of my career, I focused on the engineering aspect of building APIs, playing all the possible team roles. Later, I focused on API Design and Governance, leading API Programs, and most recently, my focus has been Developer Experience and Developer Relations because this human aspect is the most critical for a technology or product to be successful.
```